recipes/mongomock/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{%set name = "mongomock" %}
+{%set version = "3.6.0" %}
+{%set build_num = "0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "2e6b85f97b84d1797c5b0deceff72690a2e79f82c8ec410663a4175a83b8a80f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.{{ compress_type }}
+  url: https://github.com/mongomock/mongomock/archive/{{ version }}.{{ compress_type }}
```

Comment:
Did you try PyPI? Or did that not work? If so, can you please point me to a sample build?